packages/replay/test/unit/eventBuffer/EventBufferArray.test.ts

```diff
@@ -1,45 +1,53 @@
+import type { ReplayRecordingMode } from '@sentry/types';
+
 import { createEventBuffer } from './../../../src/eventBuffer';
 import { BASE_TIMESTAMP } from './../../index';
 
 const TEST_EVENT = { data: {}, timestamp: BASE_TIMESTAMP, type: 3 };
 
 describe('Unit | eventBuffer | EventBufferArray', () => {
-  it('adds events to normal event buffer', async function () {
-    const buffer = createEventBuffer({ useCompression: false });
+  for (const recordingMode of ['session', 'error'] as ReplayRecordingMode[]) {
```

Comment:
l: Can we use `describe.each` instead of the for loop here? I think this probably the more "jesty" approach. But feel free to ignore :)

Reply:
ah, you're right, I didn't think about `describe.each`, but this is better!